### DISCUSSION TITLE
[Questions] What happens when an input meet multiple rules?

### DISCUSSION THREAD

**Discussion by TATOAO** [gorules/zen#82] [Q&A]
Hi, I like Zen which has a nice UI and great functionality, but the documentation is a bit lame at this moment. I can't find related documents about this circumstance. 

For instance, I like to have a decision table:
```
Age > 20, output categories ['A', 'B']
gender == 'female', output categories ['C', 'D']
```

now when an input comes : 
{
'age': 25,
'gender': 'female'
}

The output will only give ['A', 'B']
But What I actually want is ['A', 'B', 'C', 'D'], how could I accomplish that?

**Comment by stefan-gorules** [comment]
Hi, you can achieve this using COLLECT hit policy:

<img width="1724" alt="image" src="https://github.com/gorules/zen/assets/127550877/f10de4b1-dbb6-4606-84f6-645c02162d02">
